rundeckapp/grails-app/services/rundeck/services/ExecutionService.groovy

```diff
@@ -4246,4 +4252,33 @@ class ExecutionService implements ApplicationContextAware, StepExecutor, NodeSte
             )
         }
     }
+
+    String createMisfireReport(ScheduledExecution scheduledExecution, Date misfireTime) {
+        String crontabExp = scheduledExecution.generateCrontabExression()
```

Comment:
I think instead of doing this we need to ask the jobSchedulesService.  there is currently a `nextExecutionTime` method, but we could add a `executionTimesBetween` method or something.  The reason is the schedule may not be defined on the job itself